src/store/modules/layers.store.js

```diff
@@ -213,6 +213,42 @@ const getters = {
         const layer = state.activeLayers.find((layer) => layer.id === layerId)
         return layer?.isExternal || (layer?.type === LayerTypes.KML && !layer?.adminId)
     },
+
+    /**
+     * Returns true if the layer comes from a third party (external layer or KML layer) which has
+     * been imported from a local file.
+     *
+     * KML layer are treated as external when they are generated by another user (no adminId).
+     *
+     * @param {string} layerId Layer ID of the layer to check for data disclaimer
+     * @returns {Boolean}
+     */
+    hasDataDisclaimerLocal: (state) => (layerId) => {
```

Comment:
I would maybe rename this function. Maybe simply something like
`isLocalFile`

Reply:
Might also make sense to receive directly the layer object so you don't have to do the lookup here (both usage of this function are aware of the full object)